<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HiddenGrotto</class>
 <widget class="QWidget" name="HiddenGrotto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hidden Grotto</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxProfile">
     <property name="title">
      <string>Profile</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="labelProfile">
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxProfiles"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonProfileManager">
        <property name="text">
         <string>Manager</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelProfileTID">
        <property name="text">
         <string>TID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelProfileTIDValue"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelProfileSID">
        <property name="text">
         <string>SID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelProfileSIDValue"/>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="labelProfileMACAddress">
        <property name="text">
         <string>MAC Address</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="labelProfileMACAddressValue"/>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="labelProfileDSType">
        <property name="text">
         <string>DS Type</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="labelProfileDSTypeValue"/>
      </item>
      <item row="0" column="7" rowspan="2">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="QLabel" name="labelProfileVCount">
        <property name="text">
         <string>VCount</string>
        </property>
       </widget>
      </item>
      <item row="0" column="9">
       <widget class="QLabel" name="labelProfileVCountValue"/>
      </item>
      <item row="1" column="8">
       <widget class="QLabel" name="labelProfileTimer0">
        <property name="text">
         <string>Timer0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="9">
       <widget class="QLabel" name="labelProfileTimer0Value"/>
      </item>
      <item row="0" column="10" rowspan="2">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="11">
       <widget class="QLabel" name="labelProfileGxStat">
        <property name="text">
         <string>GxStat</string>
        </property>
       </widget>
      </item>
      <item row="0" column="12">
       <widget class="QLabel" name="labelProfileGxStatValue"/>
      </item>
      <item row="1" column="11">
       <widget class="QLabel" name="labelProfileVFrame">
        <property name="text">
         <string>VFrame</string>
        </property>
       </widget>
      </item>
      <item row="1" column="12">
       <widget class="QLabel" name="labelProfileVFrameValue"/>
      </item>
      <item row="0" column="13" rowspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="14">
       <widget class="QLabel" name="labelProfileKeypresses">
        <property name="text">
         <string>Keypresses</string>
        </property>
       </widget>
      </item>
      <item row="0" column="15">
       <widget class="QLabel" name="labelProfileKeypressesValue"/>
      </item>
      <item row="1" column="14">
       <widget class="QLabel" name="labelProfileGame">
        <property name="text">
         <string>Game</string>
        </property>
       </widget>
      </item>
      <item row="1" column="15">
       <widget class="QLabel" name="labelProfileGameValue"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabRNGSelector">
     <widget class="QWidget" name="tabGenerator">
      <attribute name="title">
       <string>Generator</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxGeneratorRNGInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>RNG Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QLabel" name="labelGeneratorSeed">
            <property name="text">
             <string>Seed</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="TextBox" name="textBoxGeneratorSeed"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelGeneratorInitialAdvances">
            <property name="text">
             <string>Initial Advances</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="TextBox" name="textBoxGeneratorInitialAdvances">
            <property name="text">
             <string notr="true">0</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelGeneratorMaxAdvances">
            <property name="text">
             <string>Max Advances</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="TextBox" name="textBoxGeneratorMaxAdvances">
            <property name="text">
             <string notr="true">1000</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelGeneratorDelay">
            <property name="text">
             <string>Delay</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="TextBox" name="textBoxGeneratorDelay"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelGeneratorGrottoPower">
            <property name="text">
             <string>Grotto Power</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="ComboBox" name="comboBoxGeneratorGrottoPower">
            <item>
             <property name="text">
              <string notr="true">0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>S/MAX</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QPushButton" name="pushButtonGenerate">
            <property name="text">
             <string>Generate</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxGeneratorSettings">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <widget class="QLabel" name="labelGeneratorLocation">
            <property name="text">
             <string>Location</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ComboBox" name="comboBoxGeneratorLocation"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelGeneratorPokemon">
            <property name="text">
             <string>Pokemon</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ComboBox" name="comboBoxGeneratorPokemon"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelGeneratorItems">
            <property name="text">
             <string>Items</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="ComboBox" name="comboBoxGeneratorItems"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBoxGeneratorFilter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Filters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="labelGeneratorSlot">
            <property name="text">
             <string>Slot</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="CheckList" name="checkListGeneratorSlot">
            <item>
             <property name="text">
              <string notr="true">0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">10</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelGeneratorGroup">
            <property name="text">
             <string>Group</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="CheckList" name="checkListGeneratorGroup">
            <item>
             <property name="text">
              <string notr="true">0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">3</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelGeneratorGender">
            <property name="text">
             <string>Gender</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="CheckList" name="checkListGeneratorGender">
            <item>
             <property name="text">
              <string notr="true">♂</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>♀</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="TableView" name="tableViewGenerator"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSearcher">
      <attribute name="title">
       <string>Searcher</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_20">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxSearcherRNGInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>RNG Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QLabel" name="labelSearcherStartDate">
            <property name="text">
             <string>Start Date</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="DateEdit" name="dateEditSearcherStartDate"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelSearcherEndDate">
            <property name="text">
             <string>End Date</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="DateEdit" name="dateEditSearcherEndDate"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelSearcherMaxAdvances">
            <property name="text">
             <string>Max Advances</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="TextBox" name="textBoxSearcherMaxAdvances">
            <property name="text">
             <string notr="true">100</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelSearcherGrottoPower">
            <property name="text">
             <string>Grotto Power</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="ComboBox" name="comboBoxSearcherGrottoPower">
            <item>
             <property name="text">
              <string notr="true">0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>S/MAX</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButtonSearch">
              <property name="text">
               <string>Search</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonCancel">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QGroupBox" name="groupBoxSearcherSettings">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="labelSearcherLocation">
            <property name="text">
             <string>Location</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ComboBox" name="comboBoxSearcherLocation"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelSearcherPokemon">
            <property name="text">
             <string>Pokemon</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ComboBox" name="comboBoxSearcherPokemon"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelSearcherItems">
            <property name="text">
             <string>Items</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="ComboBox" name="comboBoxSearcherItems"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QGroupBox" name="groupBoxSearcherFilter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Filters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <widget class="QLabel" name="labelSearcherSlot">
            <property name="text">
             <string>Slot</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="CheckList" name="checkListSearcherSlot">
            <item>
             <property name="text">
              <string notr="true">0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">9</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">10</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelSearcherGroup">
            <property name="text">
             <string>Group</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="CheckList" name="checkListSearcherGroup">
            <item>
             <property name="text">
              <string notr="true">0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">3</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelSearcherGender">
            <property name="text">
             <string>Gender</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="CheckList" name="checkListSearcherGender">
            <item>
             <property name="text">
              <string notr="true">♂</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string notr="true">♀</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="4">
        <widget class="QProgressBar" name="progressBar"/>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="TableView" name="tableViewSearcher">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Form/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Form/Controls/TableView.hpp</header>
  </customwidget>
  <customwidget>
   <class>DateEdit</class>
   <extends>QDateEdit</extends>
   <header>Form/Controls/DateEdit.hpp</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>Form/Controls/ComboBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>CheckList</class>
   <extends>QComboBox</extends>
   <header>Form/Controls/CheckList.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxProfiles</tabstop>
  <tabstop>pushButtonProfileManager</tabstop>
  <tabstop>tabRNGSelector</tabstop>
  <tabstop>textBoxGeneratorSeed</tabstop>
  <tabstop>textBoxGeneratorInitialAdvances</tabstop>
  <tabstop>textBoxGeneratorMaxAdvances</tabstop>
  <tabstop>textBoxGeneratorDelay</tabstop>
  <tabstop>comboBoxGeneratorGrottoPower</tabstop>
  <tabstop>pushButtonGenerate</tabstop>
  <tabstop>comboBoxGeneratorLocation</tabstop>
  <tabstop>comboBoxGeneratorPokemon</tabstop>
  <tabstop>comboBoxGeneratorItems</tabstop>
  <tabstop>checkListGeneratorSlot</tabstop>
  <tabstop>checkListGeneratorGroup</tabstop>
  <tabstop>checkListGeneratorGender</tabstop>
  <tabstop>tableViewGenerator</tabstop>
  <tabstop>dateEditSearcherStartDate</tabstop>
  <tabstop>dateEditSearcherEndDate</tabstop>
  <tabstop>textBoxSearcherMaxAdvances</tabstop>
  <tabstop>comboBoxSearcherGrottoPower</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>comboBoxSearcherLocation</tabstop>
  <tabstop>comboBoxSearcherPokemon</tabstop>
  <tabstop>comboBoxSearcherItems</tabstop>
  <tabstop>checkListSearcherSlot</tabstop>
  <tabstop>checkListSearcherGroup</tabstop>
  <tabstop>checkListSearcherGender</tabstop>
  <tabstop>tableViewSearcher</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
